### ISSUE TITLE
Have all asynchronous requests cached using Universal Cache to support SEO

### ISSUE THREAD

**Issue by wwelling** [DSpace/dspace-angular#69]
```
The application runs XHR requests on the server & once again on the Client-side (when the application bootstraps)
Use a UniversalCache instead of regular Http, to save certain requests so they aren't re-ran again on the Client. (Example useage here)
```

https://github.com/angular/universal-starter

With redux storing a centralized state, it seems that all asynchronous requests should be put into the store and that put into Universal Cache.

**Comment by artlowel** [comment]
On [my current rest services branch](https://github.com/artlowel/dspace-angular/tree/rest-services-refactored), all async requests are indeed put in to the store. 
I'll look in to putting the store in universal cache.

**Comment by artlowel** [comment]
This was fixed by #62
